pkg/scanner/enrich.go

```diff
@@ -57,7 +57,9 @@ func (e *insightsBasedPackageEnricher) Enrich(pkg *models.Package,
 		pkg.PackageDetails.Name, pkg.PackageDetails.Version)
 
 	res, err := e.client.GetPackageVersionInsightWithResponse(context.Background(),
-		pkg.Manifest.Ecosystem, pkg.Name, pkg.Version)
+		// pkg.Manifest.Ecosystem,
```

Comment:
Please remove commented line. But are we sure we don't want to print the ecosystem in the report?

Reply:
I kept it for debugging. Current we will take ecosystem from 		string(pkg.PackageDetails.Ecosystem),